Models/AnimatedColorsCube/glTF/AnimatedColorsCube.gltf

```diff
@@ -0,0 +1,441 @@
+{
+	"asset":{
+		"copyright":"CC0: This work has been marked as dedicated to the public domain. Model by Ed Mackey.",
+		"generator":"Khronos glTF Blender I/O v4.2.57",
+		"version":"2.0"
+	},
+	"extensionsUsed":[
+		"KHR_animation_pointer"
+	],
+	"scene":0,
+	"scenes":[
+		{
+			"name":"Scene",
+			"nodes":[
+				0,
+				1,
+				2,
+				3
+			]
+		}
+	],
+	"nodes":[
+		{
+			"mesh":0,
+			"name":"TestCube",
+			"translation":[
+				-3,
+				3,
+				0
+			]
+		},
+		{
+			"mesh":1,
+			"name":"1-RedCube",
+			"translation":[
+				-3,
+				0,
+				0
+			]
+		},
+		{
+			"mesh":2,
+			"name":"2-GreenCube"
+		},
+		{
+			"mesh":3,
+			"name":"3-BlueCube",
+			"translation":[
+				3,
+				0,
+				0
+			]
+		}
+	],
+	"animations":[
+		{
+			"channels":[
+				{
+					"sampler":0,
+					"target":{
+						"node":0,
+						"path":"translation"
+					}
+				},
+				{
+					"sampler":1,
+					"target":{
+						"node":0,
+						"path":"rotation"
+					}
+				},
+				{
+					"sampler":2,
+					"target":{
+						"extensions":{
+							"KHR_animation_pointer":{
+								"pointer":"/materials/0/pbrMetallicRoughness/baseColorFactor"
+							}
+						},
+						"path":"pointer"
+					}
+				}
+			],
+			"name":"Cube Animation",
+			"samplers":[
+				{
+					"input":9,
+					"interpolation":"LINEAR",
+					"output":10
+				},
+				{
+					"input":9,
+					"interpolation":"LINEAR",
+					"output":11
+				},
+				{
+					"input":12,
+					"interpolation":"LINEAR",
+					"output":13
+				}
+			]
+		}
+	],
+	"materials":[
+		{
+			"doubleSided":true,
```

Comment:
Why doubleSided?

Reply:
Not really intentional, it's just Blender's default.  Unlike most of my samples, this one actually came out of Blender directly, with no hand-edits.

Reply:
It seems odd to default to something that is likely to run worse. Is it easy to change to not double sided?

Reply:
I was able to update this in Blender and re-export cleanly (no hand-edits).